### ISSUE TITLE
Implement _sort for Date Parameters

### ISSUE THREAD

**Issue by alexanderkiel** [samply/blaze#845]
Like `Patient.birthDate`

**Comment by sidharthramesh** [comment]
Very useful feature to have indeed!

**Comment by sidharthramesh** [comment]
I tried adding a SearchParameter for date, but that didn't work with `_sort`

**Comment by alexanderkiel** [comment]
@sidharthramesh All in R4 defined search parameters work for searching but not yet for sorting. I just have not implemented sorting, because it was no priority until now. 

If you like to look more deeply into it, search parameter definitions are taken from [search-parameters.json](https://github.com/samply/blaze/blob/master/modules/db/resources/blaze/db/search-parameters.json) at startup and indices are filled accordingly. So the indices already exist. The implementation of the search functionality of the search parameters of different types, can be found [here](https://github.com/samply/blaze/tree/master/modules/db/src/blaze/db/impl/search_param)

**Comment by sidharthramesh** [comment]
Okay got it @alexanderkiel

I would like to contribute the _sort feature. Given that the indexes are already there, where should I start looking to implement sorting?

**Comment by alexanderkiel** [comment]
@sidharthramesh Great! I think, it best to discuss this on Monday.

**Comment by alexanderkiel** [comment]
> > [@sidharthramesh](https://github.com/sidharthramesh) Great! I think, it best to discuss this on Monday.
> 
> I'm eager to know, is there any update on this discussion?

Unfortunately not. Is that a blocker for you. The priority of this issue is quite low in the moment.

**Comment by lamurian** [comment]
> > I'm eager to know, is there any update on this discussion?
> 
> Unfortunately not. Is that a blocker for you. The priority of this issue is quite low in the moment.

No, not at the moment. I'm currently working on a health survey platform on top of Blaze. It'd be nice if, at some point, we could implement sorting by date. Unfortunately, I don't have the technical capacity to contribute as a developer. I'm sharing this repo with my network, so perhaps I might be able to invite someone to contribute.
